test/infrastructure/host-cluster.sh

```diff
@@ -0,0 +1,201 @@
+# Environment Variables Config
+export CLUSTER_NAME=testing-ci-cluster-2
+export GUEST_CLUSTER_NAME=example-guest-cluster
+export AWS_PROFILE=karpenter-ci
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+export AWS_REGION=us-west-2
+export KARPENTER_VERSION=v0.10.1
+
+# Create Cluster
+cat <<EOF > cluster.yaml
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+metadata:
+  name: ${CLUSTER_NAME}
+  region: ${AWS_REGION}
+  version: "1.21"
+  tags:
+    karpenter.sh/discovery: ${CLUSTER_NAME}
+managedNodeGroups:
+  - instanceType: m5.large
+    amiFamily: AmazonLinux2
+    name: ${CLUSTER_NAME}-ng
+    desiredCapacity: 1
+    minSize: 1
+    maxSize: 10
+iam:
+  withOIDC: true
+EOF
+eksctl create cluster -f cluster.yaml
+#----------------
+# Sleep 10 in between each to allow Tekton CRDs to install.
+kubectl apply -f https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.33.2/release.yaml
+sleep 10
+kubectl apply -f https://storage.googleapis.com/tekton-releases/triggers/previous/v0.19.0/release.yaml
+sleep 10
+kubectl apply -f https://github.com/tektoncd/dashboard/releases/download/v0.24.1/tekton-dashboard-release.yaml
+sleep 10
+#----------------
+
+# AWS Load balancer
+## This will fail if the policy already exists.
+curl -o iam_policy.json https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.3.1/docs/install/iam_policy.json
+aws iam create-policy \
+    --policy-name AWSLoadBalancerControllerIAMPolicy \
```

Comment:
can you name this something else since this will conflict if there are any other `AWSLoadBalancerControllerIAMPolicy` within the whole account since IAM is global?  I think prepending or suffixing a namespace for this use-case would be good and carry it over to any other resource you're creating the test-infra. Maybe something like `${AWS_REGION}-${CLUSTER_NAME}` then this script would work to provision multiple clusters in the same account with different names.

Reply:
good idea, I'm going to name all policies `Karpenter-Policy_name`